resources/bin/bal.bat

```diff
@@ -82,7 +82,7 @@ if "%RUN_BALLERINA%" == "true" (
         set BALLERINA_HOME=%%a
     )
     if exist %userprofile%\.ballerina\ballerina-version (
-        set "FILE_PATH=%userprofile%\.ballerina\ballerina-version"
+        set FILE_PATH="%userprofile%\.ballerina\ballerina-version"
```

Comment:
This is causing the windows version test failure

Reply:
@HirumalPriyashan 
Above test failure is valid as this fix will break version switching to older versions
You can reproduce the behaviour by doing a `bal dist pull 1.2.0` and do a `bal -v` it does not show `jBallerina 1.2.0`

In that case shall we remove this and merge the PR. To verify i have sent another PR without this and tests are passing there
https://github.com/ballerina-platform/ballerina-update-tool/pull/201/files

Reply:
@HirumalPriyashan Can you check if it fails when the %userprofile% contains a space without this line?

Reply:
Since `%userprofile%` holds an unquoted value it will generate an error while the execution.